guides/installation/flex.md

```diff
@@ -0,0 +1,173 @@
+# Symfony Flex
+
+Symfony Flex is a composer plugin which helps you to manage and stay up to date with your Shopware Configuration files.
+
+## Set up a new project
+
+To create a new Shopware project just run the following command:
+
+```bash
+composer create-project shopware/production:dev-flex <project-name>
+```
+
+This will create a new project in the `<project-name>` directory. The `dev-flex` version constraint will install the latest version of the Shopware. This constraint `dev-flex` will be removed when the template is generally available.
+
+The template contains all Shopware bundles like `shopware/administration`, `shopware/storefront`, `shopware/elasticsearch`. If you don't need one of them, you can just uninstall it with:
+`composer remove shopware/<bundle-name>`.
+
+As next, you have to adjust the generated `.env` file and run following command:
+
+```bash
+bin/console system:install --basic-setup
+```
+
+This will install Shopware and create a default sales channel with a default admin user named `admin` with password `shopware`. Change these credentials after the installation.
+
+### Optional packages
+
+The template is very small and does not contain any dev-tooling or integrations like PaaS or Fastly. You can easily add them to your project with the following commands:
+
+```bash
+# Install Symfony Default Profiler
+composer req --dev profiler
+
+# Install PaaS integration
+composer req paas
+
+# Install Fastly integration
+composer req fastly
+```
+
+## How to migrate to Symfony Flex?
```

Comment:
It's called migration but it does not cover the common case of having extensions installed

Reply:
it stays same